compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -733,7 +733,10 @@ getMatmulOrIGEMMLoweringConfigAndWorkgroupSize(
   SmallVector<Attribute> promotionArray = {useGlobalDma, useGlobalDma};
   SmallVector<int64_t> promotionList = {0, 1};
   if (scaled) {
-    promotionArray.append({useGlobalDma, useGlobalDma});
+    // TODO(Max191): We don't use global load DMA for scaled matmuls, because
```

Comment:
It is better to create an issue and attach repro. Otherwise, we mistrack "compilation doesn't support it" and it is hard to remove the TODO